packages/suite/src/views/wallet/transactions/TransactionList/TransactionListActions/TransactionListActions.tsx

```diff
@@ -71,6 +76,17 @@ export const TransactionListActions = ({
         setSearch('');
     }, [account.symbol, account.index, account.accountType, setSearch]);
 
+    useEffect(() => {
```

Comment:
ok, but fill is superslow. is it possible to make it faster?

Reply:
When I was trying it the whole search was slow on like 7+ pages of transactions so coping the address manually into the input or clicking the button was quite the same.

If think this can be addressed in other PR 🙏